bricks/basic/src/popover/popover.shadow.css

```
@@ -0,0 +1,3 @@
+.popover::part(popup) {
```

在 shadow dom 里不需要额外加个 class 吧？

```suggestion
sl-popup::part(popup) {
```